cli/tests/bundle/fixture15.out

```diff
@@ -1,3 +1,3 @@
 export function getIndex(c) {
-    return "\x00\r\n\x85\u2028\u2029".indexOf(c);
+    return "\u{0}\r\n\u{85}\u{2028}\u{2029}".indexOf(c);
```

Comment:
Something is wrong if you need to do this now... @kdy1 fixed this, so it shouldn't modify this...

Reply:
See: https://github.com/denoland/deno/pull/8162#issuecomment-731719973

Reply:
Thanks for pointing this out, you're right that it shouldn't change, that said, I'm a bit lost on the cause of this change, removed code is exactly the same as code in `parse` 😬 I'll look for the problem

Reply:
Actually, it seems that `transpile_module` already passes `Rc<SourceMap>` as an argument

Reply:
Alright, I've managed to fix it by splitting `parse()` into `parse_module()` (that works as `parse()` did) and adding lower-level utility called `parse_with_source_map()`